meson.build

```diff
@@ -0,0 +1,222 @@
+project('libass', 'c', license: 'ISC',
+        meson_version: '>= 0.55.0',
+        default_options: ['c_std=c99', 'buildtype=debugoptimized', 'warning_level=2'],
+        version: '0.15.0')
+
+conf = configuration_data()
+deps = []
+
+if get_option('large-tiles')
+    conf.set('CONFIG_LARGE_TILES', 1)
+endif
+
+cc = meson.get_compiler('c')
+if cc.get_argument_syntax() == 'gcc'
+    cflags = [
+        '-Wno-sign-compare',
+        '-Wno-unused-parameter',
+        '-Werror-implicit-function-declaration',
+        '-Wstrict-prototypes',
+        '-Wpointer-arith',
+        '-Wredundant-decls',
+        '-Wno-missing-field-initializers'
+    ]
+    add_project_arguments(cc.get_supported_arguments(cflags), language: 'c')
+endif
+link_args = cc.get_supported_link_arguments(['-prefer-non-pic'])
+
+check_functions = [
+    'strdup',
+    'strndup'
+]
+
+m_dep = cc.find_library('m', required: false)
+if m_dep.found()
+    deps += m_dep
+endif
+
+iconv_dep = cc.find_library('iconv', required: false)
+if iconv_dep.found()
+    deps += iconv_dep
+endif
+if iconv_dep.found() or cc.has_function('iconv_open')
+    conf.set('CONFIG_ICONV', 1)
+endif
+
+# ASM
+nasm_args = []
+nasm = find_program('nasm', required: get_option('asm'))
+enable_asm = nasm.found()
+
+if enable_asm
+    # NASM version check
+    nasm_r = run_command(nasm, '-v')
+    out = nasm_r.stdout().strip().split()
+    if out[1].to_lower() == 'version'
+        if out[2].version_compare('< 2.10')
```

Comment:
Will this check work with custom built non-release versions? Eg `NASM version 2.16rc0 compiled on Feb 25 2021`
*(I assume you did verify `2.10` is the first versionsupporting `vpmovzxwd`)*

Reply:
No clue re: custom-built, you'd have to try. I also think I verified that but no promises, it's just been way too long.

Reply:
Looks like 2.10 is when they added support for AVX2.

Reply:
Looks like it works.

meson.build:
```
message('Version check test')
message('2.16rc0'.version_compare('> 2.10'))
```

Output:
```
Message: Version check test
Message: True
```